### DISCUSSION TITLE
List folders

### DISCUSSION THREAD

**Discussion by Octy1003** [community/community#113210] [Other Feature Feedback, Questions, & Ideas]
It would be great if we could make different "folders" that we can store our lists in.

For example categorizing a FOSS collection:

* Linux folde:
  - Everyday tools list 
  - Productivity list
  - ...

* Windows folder:
  - Everyday tools list
  - ...

**Comment by adnangujjar98** [comment]
> It would be great if we could make different "folders" that we can store our lists in.
> 
> For example categorizing a FOSS collection:
> 
> * Linux folde:
>   - Everyday tools list 
>   - Productivity list
>   - ...
> 
> * Windows folder:
>   - Everyday tools list
>   - ...
> 
>  

👍

**Comment by naseroish** [comment]
So you mean like having subfolders within the list like:
 
- Main Folder
  - Sub Folder 
    - Starred Repository
- Other Main Folder
....
 
If this what you mean it would be great all i see in list right now is just ability to create one folder you cant nest folders inside each other
